tests/Cluster/ClusterInfoTest.cpp

```diff
@@ -120,3 +121,53 @@ TEST_F(ClusterInfoTest, testServerAliasExists) {
   }
 }
 
+TEST_F(ClusterInfoTest, plan_will_provide_latest_id) {
+  auto &cache {server.getFeature<ClusterFeature>().agencyCache()};
+  auto [acb, index] {cache.read({AgencyCommHelper::path("Sync/LatestID")})};
+  auto expectedLatestId {acb->slice().at(0).get("arango").get("Sync").get("LatestID").getInt()};
```

Comment:
Note that this can throw, which will then break the test. So it is OK here, but in actual production code we need to be more careful.

Reply:
The variant `get(std::vector<std::string>)` does not throw if the top level is an object, though, it simply returns a `NoneSlice` if someway on the path an attribute is not found.